### ISSUE TITLE
Exception thrown from `connect_failed`

### ISSUE THREAD

**Issue by grunskis** [getsenic/nuimo-linux-python#28]
Traceback:

```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 230, in maybe_handle_message
    self._handler(*args, **kwargs)
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 152, in _interfaces_added
    self._device_discovered(path, interfaces)
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 166, in _device_discovered
    self.device_discovered(device)
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/nuimo/nuimo.py", line 49, in device_discovered
    self.listener.controller_discovered(device)
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/senic_hub/backend/nuimo_setup.py", line 89, in controller_discovered
    self._controller.connect()
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/nuimo/nuimo.py", line 125, in connect
    super().connect()
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 253, in connect
    self._connect()
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 270, in _connect
    self._connect()
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 270, in _connect
    self._connect()
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 270, in _connect
    self._connect()
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 270, in _connect
    self._connect()
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/gatt/gatt_linux.py", line 276, in _connect
    self.connect_failed(_error_from_dbus_error(e))
  File "/srv/senic_hub/venv/lib/python3.4/site-packages/nuimo/nuimo.py", line 132, in connect_failed
    super.connect_failed(error)
AttributeError: type object 'super' has no attribute 'connect_failed'
```

**Comment by Charliedean** [comment]
Also having this issue. Did you manage to fix?

**Comment by larsblumberg** [comment]
Fixed in [latest release 0.3.2](https://github.com/getsenic/nuimo-linux-python/releases/tag/0.3.2)
